FF128Relnote: RTCRtpTransceiver.setCodecPreferences() supported (#34144)
mdn/content: files/en-us/mozilla/firefox/releases/128/index.md

### APIs

> - {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).

Author: ```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec — including those used for retransmission, redundancy, and forward error correction — and to disable the negotiation of specific codecs ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

### APIs

> <<<<<<< HEAD
> <<<<<<< HEAD

Author: ```suggestion
```

- {{domxref('Blob.bytes()')}} is supported for returning the data from a {{domxref('Blob')}} as an array of bytes in a {{jsxref("Uint8Array")}} object. ([Firefox bug 1896509](https://bugzil.la/1896509)).
- {{domxref('MediaKeys.getStatusForPolicy()')}} is now supported for checking whether the CDM module, which is used to decrypt DRM protected content, would allow the presentation of encrypted media data for a "hypothetical" key based on specified policy requirements such as the [High-bandwidth Digital Content Protection (HDCP)](https://en.wikipedia.org/wiki/High-bandwidth_Digital_Content_Protection) version supported by the system.
  This provides an application with a simple mechanism to know in advance whether playback at the optimal resolution will be allowed, without having to create a media key session or fetch a real license. ([Firefox bug 1878714](https://bugzil.la/1878714)).

> - {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preferred order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).

Author: @bsmth FYI only The blog link you pointed me to has inspired me to look at this again and make this a little more clear.

```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the codecs that a WebRTC local peer is able to use for decoding received data, in its preferred codec order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

> - {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preference order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).

Reviewer: Minor reword, what do you think?

```suggestion
- {{domxref('RTCRtpTransceiver.setCodecPreferences()')}} is now supported for setting the allowed codecs in preferred order. Web applications can use this to cause the remote peer to choose a preferred codec, and to disable the negotiation of specific codecs — including those used for retransmission, redundancy, and forward error correction. ([Firefox bug 1396922](https://bugzil.la/1396922)).
```

Author: Thanks. I'm ambivalent so I'll accept it :-)
